src/routes/index.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
```

Comment:
This file has been deleted already. You should remove it

Reply:
All right 

Removed